app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Is there a specific spot where you think we should be using these? A scope for active games would be good, since this involves some `OR` logic, but I don't want to go overboard with setting up scopes for situations that can easily be handled with `where(property: value)`.

Btw, I'm planning to set up all the connections between the three tables as part of a separate ticket: https://trello.com/c/Srt3LB1I/32-create-connections-between-the-user-game-and-piece-models